libr/anal/var.c

```diff
@@ -663,11 +663,46 @@ static void extract_arg(RAnal *anal, RAnalFunction *fcn, RAnalOp *op, const char
 	free (esil_buf);
 }
 
+static bool is_used_like_arg(const char *regname, const char *opsreg, const char *opdreg, RAnalOp *op) {
+	#define STR_EQUAL(s1, s2) s1 && s2 && !strcmp (s1, s2)
```

Comment:
we have r_str_cmp() which does this already, remove this macro

Reply:
It doesn't. It returns 0 when both are NULL. I need it to return true only if they aren't NULLs and equal.

Reply:
Correct..   not sure if we should change rstrcmp to behave like this.. @ret2libc